### ISSUE TITLE
Training not stable on custom dataset

### ISSUE THREAD

**Issue by htzheng** [NVlabs/stylegan2-ada-pytorch#124]
When training on Place2 dataset, I sometimes observe sudden spikes in the loss curve as shown in the figure. I used pl and R1 regularization,
![image](https://user-images.githubusercontent.com/8148851/121792062-950fd680-cbbe-11eb-8362-5fe367b81338.png)

I wonder is there a way to stabilize the training, many thanks!

**Comment by thusinh1969** [comment]
Same here. My Fid50 was 78 and suddenly went bizarre to 393... ?!?! Any hint? Should I stop and train all over again ?

Steve
